RoslynPluginGenerator/RoslynPluginJarBuilder.cs

```diff
@@ -315,20 +314,6 @@ private string FindPluginKey()
             return pluginKey;
         }
 
-        /// <summary>
-        /// Sets the invariant, required manifest properties
-        /// </summary>
-        private void SetFixedManifestProperties()
-        {
-            // This property must appear first in the manifest.
-            // See http://docs.oracle.com/javase/6/docs/technotes/guides/jar/jar.html#JAR%20Manifest
-            jarManifestBuilder.SetProperty("Sonar-Version", "4.5.2");
```

Comment:
This was the bug - the SDK was hard-coding values that should have been copied from the template jar.